src/pyodide-py/pyodide/webloop.py

```diff
@@ -156,6 +156,20 @@ def call_at(
         delay = when - cur_time
         return self.call_later(delay, callback, *args, context=context)
 
+    def run_in_executor(self, executor, func, *args):
+        """This is normally supposed to run func(args) in a separate thread and
```

Comment:
Could we have a 1 line summary, then an empty line? :)

It's not just numpydoc conversion also general Python convention for docstrings,

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line.